test/functional/api/cas/cli_messages.py

```diff
@@ -252,23 +252,24 @@
     r"Option '--cache-line-size \(-x\)' is not allowed"
 ]
 
-def check_stderr_msg(output: Output, expected_messages, negate=False):
-    return __check_string_msg(output.stderr, expected_messages, negate)
 
-
-def check_stdout_msg(output: Output, expected_messages, negate=False):
-    return __check_string_msg(output.stdout, expected_messages, negate)
-
-
-def __check_string_msg(text: str, expected_messages, negate=False):
+def check_string_msg_all(text: str, expected_messages):
     msg_ok = True
     for msg in expected_messages:
         matches = re.search(msg, text)
-        if not matches and not negate:
+        if not matches:
             TestRun.LOGGER.error(f"Message is incorrect, expected: {msg}\n actual: {text}.")
             msg_ok = False
-        elif matches and negate:
-            TestRun.LOGGER.error(f"Message is incorrect, expected to not find: {msg}\n "
-                                 f"actual: {text}.")
-            msg_ok = False
+    return msg_ok
+
+
+def check_string_msg_any(text: str, expected_messages):
+    msg_ok = False
+    for msg in expected_messages:
+        matches = re.search(msg, text)
+        if matches:
+            msg_ok = True
+            break
+    if not msg_ok:
+        TestRun.LOGGER.error(f"Message is incorrect: {text}.")
```

Comment:
`check_string_msg_all()` and `check_string_msg_any()` are perfect opportunities to use python's `all()` and `any()`. Consider the following:
```
def check_string_msg_all(text: str, expected_messages):
    if all([re.match(m, text) for m in expected_messages]):
        return True
     
    TestRun.LOGGER.error(f"Message is incorrect, expected: {msg}\n Got: {text}.")
    return False
```
and 
```
def check_string_msg_any(text: str, expected_messages):
    if any([re.match(m, text) for m in expected_messages]):
        return True
     
    TestRun.LOGGER.error(f"Message is incorrect, expected at least one of: {msg}\n Got: {text}.")
    return False
```